cmd/main.go

```diff
@@ -58,24 +66,6 @@ func init() {
 }
 
 func runMain() int {
```

Comment:
Instead of returning an error code, why not just return the error? `main()` will be authoritative and decide the exit code based on the error. It'll just be this

```go
	if err := runMain(); err != nil {
		os.Exit(1)
	}
        os.Exit(0)
```